Software/Development/SemesterProject/System/scheduler.c

```diff
@@ -100,7 +100,7 @@ void task100us(void)
 void task50us(void)
 {
     executeControl();
-    monitorErrorSources();
+    isErrorMonitoringEnabled();
```

Comment:
The name of the function would be a great name if it returned a boolean which explained whether the error monitoring software component is enabled. However it should explain that is the most upstream function of the error monitoring.

Reply:
Nicolai nicolai..... I really expected more...

Reply:
🤷‍♂🤷‍♂